static/js/src/ua-payment-modal.js

```diff
@@ -431,6 +401,43 @@ function applyGuestPurchaseTotals() {
   setOrderTotals(null, false, purchaseTotals, modal);
 }
 
+function fetchCustomerInfo(accountId) {
+  getCustormerInfo(accountId)
+    .then((res) => {
+      const { name, address } = res.customerInfo;
+      customerInfo = { ...customerInfo, name, address };
+      setFormElements();
+      isCustomerInfoSet = true;
+    })
+    .catch((e) => console.error(e));
```

Comment:
Do we have sentry configured on ubuntu.com? Might nice to send an error here.

Reply:
Yeah, Sentry is setup on the backend at least - this call hits that, which then calls the contracts API and raises any errors at that point, to be captured by Sentry.